pulpcore/cli/common/__init__.py

```diff
@@ -42,6 +42,10 @@ def _config_callback(ctx: click.Context, param: Any, value: Optional[str]) -> No
 @click.option("--base-url", default="https://localhost", help="Api base url")
 @click.option("--username", help="Username on pulp server")
 @click.option("--password", help="Password on pulp server")
+@click.option("--client-cert", help="Path to client certificate")
+@click.option(
+    "--client-key", help="Path to client private key. Not required if client-cert contains this."
+)
```

Comment:
I'm tempted to use `--cert` and `--key` here since that's the options curl uses.

Reply:
Would work for me.

Reply:
Now you missed those two appearances.